### ISSUE TITLE
Payload 3

### ISSUE THREAD

**Issue by christopherpickering** [brachypelma/payload-plugin-comments#4]
Thanks for building this. When I'm trying to run in payload 3, there is an error when generating types: TypeError: comments is not a function.

**Comment by mcmadafly** [comment]
@christopherpickering - any resolution on this?

**Comment by christopherpickering** [comment]
no, it just needs to be converted from a v2 pluggin to a v3. I didn't finish the project I was planning for it yet, when we do, maybe we will be able to help with the update.
